src/KingSystem/World/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+target_sources(uking PRIVATE
+  worldMgr.h
```

Comment:
worldManager would be preferable. Nintendo would call it "WorldMgr" or "world::Manager" but not worldMgr